client/components/fraud-risk-tools-banner/index.tsx

```diff
@@ -27,29 +24,15 @@ const FRTDiscoverabilityBanner: React.FC = () => {
 		try {
 			return JSON.parse( frtDiscoverBannerSettings );
 		} catch ( e ) {
-			return { remindMeCount: 0, remindMeAt: null, dontShowAgain: false };
+			return { dontShowAgain: false };
 		}
 	} );
 
-	const showBanner =
-		! settings.dontShowAgain &&
-		( null === settings.remindMeAt || Date.now() > settings.remindMeAt );
-
-	const setReminder = () => {
-		const nowTimestamp = Date.now();
-		setSettings( ( prevSettings ) => {
-			return {
-				...prevSettings,
-				remindMeCount: prevSettings.remindMeCount + 1,
-				remindMeAt: nowTimestamp + 3 * TIME.DAY_IN_MS,
-			};
-		} );
-	};
+	const showBanner = ! settings.dontShowAgain;
 
 	const setDontShowAgain = () => {
-		setSettings( ( prevSettings ) => {
+		setSettings( () => {
```

Comment:
You can replace the callback function syntax by:
```js
setSettings( { dontShowAgain: true } );
```

Reply:
Thanks for flagging this. It looks a bit different with my changes based on your other recommendation. Change made in [ab8055f](https://github.com/Automattic/woocommerce-payments/pull/6664/commits/ab8055f5a75869456562f006d840b4a310fe5402).